fitlins/interfaces/bids.py

```diff
@@ -0,0 +1,90 @@
+import os
+from nipype.interfaces.base import (
+    BaseInterfaceInputSpec, TraitedSpec, SimpleInterface,
+    InputMultiPath, File, Directory,
+    traits, isdefined
+    )
+from bids import grabbids as gb, analysis as ba
+
+
+class LoadLevel1BIDSModelInputSpec(BaseInterfaceInputSpec):
+    bids_dirs = InputMultiPath(Directory(exists=True),
+                               mandatory=True,
+                               desc='BIDS dataset root directories')
+    model = File(exists=True, desc='Model filename')
+    selectors = traits.Dict(desc='Limit collected sessions')
+
+
+class LoadLevel1BIDSModelOutputSpec(TraitedSpec):
+    session_info = traits.List(traits.Dict())
+    contrast_info = traits.List(traits.Any())
+
+
+class LoadLevel1BIDSModel(SimpleInterface):
+    input_spec = LoadLevel1BIDSModelInputSpec
+    output_spec = LoadLevel1BIDSModelOutputSpec
+
+    def _run_interface(self, runtime):
+        layout = gb.BIDSLayout(self.inputs.bids_dirs)
```

Comment:
@adelavega @tyarkoni Given your need to specify `Layout` inputs, you may want to chime in on what inputs this `LoadLevel1BIDSModel` interface should have.

Reply:
Will it be necessary to have both `include` and `exclude`? Or would `inputs.include_pattern` (or `ignore_pattern`, or whatever) do what you need?

I'm fine with putting both in; I just don't want to add options that aren't going to be used.